web_sdk/web_test_utils/lib/skia_client.dart

```diff
@@ -279,6 +288,40 @@ class SkiaGoldClient {
     }
   }
 
+  List<String> _getMatchingArguments(String testName, int screenshotSize) {
+    // The algorithm to be used when matching images. The available options are:
+    // - "fuzzy": Allows for customizing the thresholds of pixel differences.
+    // - "sobel": Same as "fuzzy" but performs edge detection before performing
+    //            a fuzzy match.
+    const String algorithm = 'fuzzy';
+
+    // The number of pixels in this image that are allowed to differ from the
+    // baseline. It's okay for this to be a slightly high number like 10% of the
+    // image size because those wrong pixels are constrained by
+    // `pixelDeltaThreshold` below.
+    final int maxDifferentPixels = (screenshotSize * kMaxDifferentPixelsRate).toInt();
+
+    // The maximum acceptable difference in RGB channels of each pixel.
+    //
+    // ```
+    // abs(r(image) - r(golden)) + abs(g(image) - g(golden)) + abs(b(image) - b(golden)) <= pixelDeltaThreshold
+    // ```
+    final String pixelDeltaThreshold;
+    if (testName.startsWith('canvaskit_')) {
```

Comment:
Test name starting with 'canvaskit_' is not guaranteed. This may lead to surprises later.

Reply:
Agreed. I'll  change this to use the boolean that indicates whether canvaskit is enabled or not.